src/utils/notify/index.js

```diff
@@ -0,0 +1,35 @@
+const admin = require('firebase-admin');
+const { query: Hasura } = require('../hasura');
+const { getFcmToken } = require('./queries');
+
+const notify = (entityTypeId, entityId, actorId, notifierIds) => {
```

Comment:
Here, the notifierIds is an Array. So please update LINE 7 to use `_in` operator instead of `_eq`